tests/test_data/exodus-config.json

```diff
@@ -2,159 +2,197 @@
+            "dest": "/content/beta/rhel",
+            "exclude_paths": []
         },
         {
             "src": "/content/beta/rhs/rhui",
-            "dest": "/content/beta/rhs"
+            "dest": "/content/beta/rhs",
+            "exclude_paths": []
         },
         {
             "src": "/content/dist/layered/rhui",
-            "dest": "/content/dist/layered"
+            "dest": "/content/dist/layered",
+            "exclude_paths": []
         },
         {
             "src": "/content/beta/layered/rhui",
-            "dest": "/content/beta/layered"
+            "dest": "/content/beta/layered",
+            "exclude_paths": []
         },
         {
             "src": "/content/dist/middleware/rhui",
-            "dest": "/content/dist/middleware"
+            "dest": "/content/dist/middleware",
+            "exclude_paths": []
         },
         {
             "src": "/content/dist/rhel8/rhui",
-            "dest": "/content/dist/rhel8"
+            "dest": "/content/dist/rhel8",
+            "exclude_paths": []
         },
         {
             "src": "/content/dist/rhel-alt/rhui",
-            "dest": "/content/dist/rhel-alt"
+            "dest": "/content/dist/rhel-alt",
+            "exclude_paths": []
         },
         {
             "src": "/content/dist/rhel/rhui",
-            "dest": "/content/dist/rhel"
+            "dest": "/content/dist/rhel",
+            "exclude_paths": []
         },
         {
             "src": "/content/dist/rhes/rhui",
-            "dest": "/content/dist/rhes"
+            "dest": "/content/dist/rhes",
+            "exclude_paths": []
         },
         {
             "src": "/content/dist/rhs/rhui",
-            "dest": "/content/dist/rhs"
+            "dest": "/content/dist/rhs",
+            "exclude_paths": []
         },
         {
             "src": "/content/e4s/rhel8/rhui",
-            "dest": "/content/e4s/rhel8"
+            "dest": "/content/e4s/rhel8",
+            "exclude_paths": []
         },
         {
             "src": "/content/e4s/rhel/rhui",
-            "dest": "/content/e4s/rhel"
+            "dest": "/content/e4s/rhel",
+            "exclude_paths": []
         },
         {
             "src": "/content/els/rhel/rhui",
-            "dest": "/content/els/rhel"
+            "dest": "/content/els/rhel",
+            "exclude_paths": []
         },
         {
             "src": "/content/eus/rhel8/rhui",
-            "dest": "/content/eus/rhel8"
+            "dest": "/content/eus/rhel8",
+            "exclude_paths": []
         },
         {
             "src": "/content/eus/rhel/rhui",
-            "dest": "/content/eus/rhel"
+            "dest": "/content/eus/rhel",
+            "exclude_paths": []
         },
         {
             "src": "/content/rc/rhel/rhui",
-            "dest": "/content/rc/rhel"
+            "dest": "/content/rc/rhel",
+            "exclude_paths": []
         }
     ],
     "releasever_alias": [
```

Comment:
Perhaps leave a few `releasever_alias`es without an `exclude_paths` key so we can test that the CDN can handle alias resolutions when an alias does not contain an exclude_paths key (i.e., the current state of all aliases on the CDN).